pallets/dia-oracle/rpc/src/lib.rs

```diff
@@ -75,12 +75,12 @@ where
 		at: Option<<Block as BlockT>::Hash>,
 	) -> RpcResult<CoinInfo> {
 		let api = self.client.runtime_api();
-		let at = BlockId::hash(at.unwrap_or_else(||
+		let at = at.unwrap_or_else(||
```

Comment:
Why is the `BlockId::hash` not required anymore?

Reply:
removed as well. https://github.com/paritytech/substrate/pull/13255